app/v2/inbound_sms/get_inbound_sms.py

```diff
@@ -0,0 +1,83 @@
+import uuid
+
+from flask import jsonify, request, url_for, current_app
+from sqlalchemy.orm.exc import NoResultFound
+from werkzeug.exceptions import abort
+
+from notifications_utils.recipients import validate_and_format_phone_number
+from notifications_utils.recipients import InvalidPhoneError
+
+from app import authenticated_service
+from app.dao import inbound_sms_dao
+from app.v2.errors import BadRequestError
+from app.v2.inbound_sms import v2_inbound_sms_blueprint
+
+
+@v2_inbound_sms_blueprint.route("/<user_number>", methods=['GET'])
+def get_inbound_sms_by_number(user_number):
+    _data = request.args.to_dict(flat=False)
+
+    # flat=False makes everything a list, but we only ever allow one value for "older_than"
+    if 'older_than' in _data:
```

Comment:
Is it easier to just use request.args.get('older_than')?

Reply:
sure, was just copy and pasting from another function

Reply:
Fair enough - I think the other function could just use the get as well